packages/frontend/src/pages/admin/server-rules.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+<div>
+	<MkStickyContainer>
+		<template #header><XHeader :tabs="headerTabs"/></template>
+		<MkSpacer :content-max="700" :margin-min="16" :margin-max="32">
+			<div class="_gaps_m">
+				<div>{{ i18n.ts._serverRules.description }}</div>
+				<Sortable
+					v-model="serverRules"
+					class="_gaps_m"
+					:item-key="(_, i) => i"
+					:animation="150"
+					:handle="'.' + $style.itemHandle"
+					@start="e => e.item.classList.add('active')"
+					@end="e => e.item.classList.remove('active')"
+				>
+					<template #item="{element,index}">
+						<div :class="$style.item">
+							<div :class="$style.itemHeader">
+								<div :class="$style.itemNumber" v-text="String(index + 1)"/>
+								<span :class="$style.itemHandle"><i class="ti ti-menu"/></span>
+								<button class="_button" :class="$style.itemRemove" @click="remove(index)"><i class="ti ti-x"></i></button>
+							</div>
+							<MkInput v-model="serverRules[index]"/>
+						</div>
+					</template>
+				</Sortable>
+				<div :class="$style.commands">
+					<MkButton rounded @click="serverRules.push('')"><i class="ti ti-plus"></i> {{ i18n.ts.add }}</MkButton>
+					<MkButton primary rounded :class="$style.buttonSave" @click="save"><i class="ti ti-check"></i> {{ i18n.ts.save }}</MkButton>
+				</div>
+			</div>
+		</MkSpacer>
+	</MkStickyContainer>
+</div>
+</template>
+
+<script lang="ts" setup>
+import { defineAsyncComponent } from 'vue';
+import XHeader from './_header_.vue';
+import * as os from '@/os';
+import { fetchInstance, instance } from '@/instance';
+import { i18n } from '@/i18n';
+import { definePageMetadata } from '@/scripts/page-metadata';
+import MkButton from '@/components/MkButton.vue';
+import MkInput from '@/components/MkInput.vue';
+
+const Sortable = defineAsyncComponent(() => import('vuedraggable').then(x => x.default));
+
+let serverRules: string[] = $ref(instance.serverRules);
+
+const save = async () => {
+	await os.apiWithDialog('admin/update-meta', {
+		serverRules,
+	});
+	fetchInstance();
+};
+
+const remove = (index: number): void => {
+	serverRules.splice(index, 1);
+};
+
+const headerTabs = $computed(() => []);
+
+definePageMetadata({
+	title: i18n.ts.serverRules,
+	icon: 'ti ti-checkbox',
+});
+</script>
+
+<style lang="scss" module>
+.item {
+	display: block;
+	color: var(--navFg);
+}
+
+.itemHeader {
+	display: flex;
+	margin-bottom: 8px;
+	align-items: center;
+}
+
+.itemHandle {
+	display: flex;
+	width: 40px;
+	height: 40px;
+	align-items: center;
+	justify-content: center;
+	cursor: move;
+}
+
+.itemNumber {
+	display: flex;
+	background-color: var(--accent);
+	color: var(--fgOnAccent);
```

Comment:
コンパネの方も accentedBg + accent にしたい
```suggestion
	background-color: var(--accentedBg);
	color: var(--accent);
```
<img width="465" alt="image" src="https://user-images.githubusercontent.com/7106976/232459144-383c4c2f-f534-4419-9b54-94829d14420c.png">